seedpod_ground_risk/layers/fatality_risk_layer.py

```diff
@@ -4,13 +4,22 @@
 from seedpod_ground_risk.layers.blockable_data_layer import BlockableDataLayer
 from seedpod_ground_risk.layers.strike_risk_layer import StrikeRiskLayer
 from seedpod_ground_risk.path_analysis.harm_models.fatality_model import FatalityModel
+from seedpod_ground_risk.ui_resources.aircraft_options import AIRCRAFT_LIST
 
 
 class FatalityRiskLayer(BlockableDataLayer):
 
-    def __init__(self, key, colour: str = None, blocking=False, buffer_dist=0, **kwargs):
+    def __init__(self, key, ac: str = 'Default',
```

Comment:
Nope, all these attributes don't do anything in `FatalityRiskLayer`, so shouldn't be here. It is unnecessary duplication of state that will cause confusion down the line when values disagree... The right way of doing this would be to add properties that just return the value in the underlying values from `self._strike_layer`:

```python
@property
def ac(self):
    return self._strike_layer.ac
```